cmd/crypto/help.go

```diff
@@ -38,29 +38,64 @@ var (
 		},
 		config.HelpKV{
 			Key:         KMSVaultAppRoleID,
-			Description: `unique role ID for approle`,
+			Description: `AppRole Role ID used to authenticate to Vault`,
 			Type:        "string",
 		},
 		config.HelpKV{
 			Key:         KMSVaultAppRoleSecret,
-			Description: `unique secret ID for approle`,
+			Description: `AppRole Secret ID used to authenticate to Vault`,
 			Type:        "string",
 		},
 		config.HelpKV{
 			Key:         KMSVaultNamespace,
-			Description: `optional KMS namespace e.g. "customer1"`,
+			Description: `optional Vault namespace - e.g. "customer1"`,
 			Optional:    true,
 			Type:        "string",
 		},
 		config.HelpKV{
 			Key:         KMSVaultKeyVersion,
-			Description: `optional key version number`,
+			Description: `optional key version if not the latest should be used`,
 			Optional:    true,
 			Type:        "number",
 		},
 		config.HelpKV{
 			Key:         KMSVaultCAPath,
-			Description: `optional path to PEM-encoded CA certs e.g. "/home/user/custom-certs"`,
+			Description: `path to PEM-encoded CA certs to verify Vault's TLS certificate - e.g. "/home/user/custom-root-CAs"`,
+			Optional:    true,
+			Type:        "path",
+		},
+		config.HelpKV{
+			Key:         config.Comment,
+			Description: config.DefaultComment,
+			Optional:    true,
+			Type:        "sentence",
+		},
+	}
+
+	HelpKeys = config.HelpKVS{
+		config.HelpKV{
+			Key:         KMSKeysEndpoint,
+			Description: `API endpoint - e.g. "https://key-endpoint:7373"`,
+			Type:        "url",
+		},
+		config.HelpKV{
+			Key:         KMSKeysKeyName,
+			Description: `key at the key server used to encrypt objects - e.g. "my-minio-key"`,
+			Type:        "string",
```

Comment:
This help string should be `unique key name`

Reply:
And for what is it used for?! If I don't know that the key name is, the help `unique key name` is kind of not helpful?

Reply:
You don't need to know what is the key used for it is a key unique on the keys server. Of course KMS itself is for encrypting objects here so we don't need to double explain that.

Reply:
If that ever changes where we use keys for anything other than encryption of objects we can become more explicit until that time, the help should be concise.